src/index.ts

```diff
@@ -233,3 +233,30 @@ class PromiseDelegate<T> {
   private _resolve: (value?: T | Thenable<T>) => void;
   private _reject: (reason?: any) => void;
 }
+
+
+/**
+ * Get global configuration data for the Jupyter application.
+ *
+ * @param name - The name of the configuration option.
+ *
+ * @returns The config value or `undefined` if not found.
+ *
+ * #### Notes
+ * For browser based applications, it is assumed that the page HTML
+ * includes a script tag with the id `jupyterConfigData` containing the
+ * configuration as valid JSON.
+ */
+export
+function getConfigOption(name: string): any {
+  if (typeof window === 'undefined') {
+    // TODO: support node here.
+    return void 0;
+  }
+  let el = document.getElementById('jupyterConfigData');
+  if (!el) {
+    return void 0;
+  }
+  let config = JSON.parse(el.textContent);
+  return (config as any)[name];
```

Comment:
If the config is nested, we'd have to make a stringify/parse before giving it away, but I suppose that is cheaper than parsing the whole thing every time.